packages/accordion/partials/AccordionItems/style.module.css

```diff
@@ -60,7 +60,7 @@
   transition: transform 0.4s;
   transform: var(--icon-transform);
 
-  & svg {
+  & img {
```

Comment:
Since this is no longer an svg, the `fill` and `stroke` styles below aren't applied and the color is changed. I know you mentioned we are accepting slight rendering differences in the PR description so if this is one of those disregard!

Reply:
Honestly, I think this is fine. We are deprecating these components in favor of the hds package now anyways.